bitbots_ball_filter/scripts/ball_sim.py

```diff
@@ -8,7 +8,9 @@
 
 class SimBall:
     def __init__(self):
-        rospy.init_node('ball_sim')
+        rclpy.init(args=sys.argv)
```

Comment:
Missing `rclpy.spin()`, so callback/time/... do not work. See `ball_publisher.py` for correct usage.